package.json

```diff
@@ -33,7 +33,8 @@
     "test:hosting-rewrites": "bash ./scripts/hosting-tests/rewrites-tests/run.sh",
     "test:triggers-end-to-end": "bash ./scripts/triggers-end-to-end-tests/run.sh",
     "test:storage-deploy": "bash ./scripts/storage-deploy-tests/run.sh",
-    "test:storage-emulator-integration": "bash ./scripts/storage-emulator-integration/run.sh"
+    "test:storage-emulator-integration": "bash ./scripts/storage-emulator-integration/run.sh",
+    "test:functions-deploy": "bash ./scripts/functions-deploy-tests/run.sh"
```

Comment:
These tests are more-or-less alphabetical order. Could you continue that (and fix it - `import-export` is in the wrong place too)